### ISSUE TITLE
[BUG REPORT] Error when checking for updates

### ISSUE THREAD

**Issue by JustDoom** [retrooper/packetevents#272]
**Describe the bug**
Error in console
```
[06:38:07 WARN]: java.lang.NumberFormatException: For input string: "v1"
[06:38:07 WARN]:        at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
[06:38:07 WARN]:        at java.base/java.lang.Integer.parseInt(Integer.java:668)
[06:38:07 WARN]:        at java.base/java.lang.Integer.parseInt(Integer.java:786)
[06:38:07 WARN]:        at FlappyAC-1.0.0.jar//com.github.retrooper.flappyac.packetevents.util.PEVersion.<init>(PEVersion.java:56)
[06:38:07 WARN]:        at FlappyAC-1.0.0.jar//com.github.retrooper.flappyac.packetevents.util.updatechecker.UpdateChecker.checkForUpdate(UpdateChecker.java:62)
[06:38:07 WARN]:        at FlappyAC-1.0.0.jar//com.github.retrooper.flappyac.packetevents.util.updatechecker.UpdateChecker.lambda$handleUpdateCheck$0(UpdateChecker.java:111)
[06:38:07 WARN]:        at java.base/java.lang.Thread.run(Thread.java:833)
[06:38:07 INFO]: [packetevents] Something went wrong while checking for an update. Your build: (2.0.0)
[06:38:07 INFO]: [packetevents] [Checking for an update again in 80 seconds...
```

**To Reproduce**
Launch plugin with packetevents 2.0

**Expected behavior**
no error

**About your server (please complete the following information):**
Purpur 1.18.1

**Comment by retrooper** [comment]
Ah, fixed this in v1.8.1 packetevents, but forgot its broken in 2.0 too.

**Comment by retrooper** [comment]
Fixed in https://github.com/retrooper/packetevents/commit/31d572a8b69916a7f228051cb206dc396758b6fe
